WearTilesKotlin/app/build.gradle

```
@@ -53,18 +49,30 @@ android {
     kotlinOptions {
         jvmTarget = "1.8"
     }
+
+    buildFeatures {
+        compose true
+    }
+
+    composeOptions {
+        kotlinCompilerExtensionVersion "${libs.versions.compose.get()}"
```

remove quotes and template.
```suggestion
        kotlinCompilerExtensionVersion libs.versions.compose.get()
```